src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_fr.properties

```diff
@@ -16,8 +16,8 @@ equals.noEquals=La définition de la méthode ''hashCode()'' doit toujours être
 equals.noHashCode=La définition de la méthode ''equals()'' doit toujours être accompagnée de la définition de la méthode ''hashCode()''.
 equalsIgnoreCase.avoid.null=Les chaînes littérales devraient être sur le côté gauche d''une comparaison ''equalsIgnoreCase''.
 explicit.init=L''initialisation explicite de la variable ''{0}'' à la valeur ''{1}'' est inutile, c''est la valeur par défaut pour ce type.
-fall.through=Le cas précédent du "switch" ne contient pas de break, return, throw ou continue.
-fall.through.last=Le dernier cas du "switch" ne contient pas de break, return, throw ou continue.
+fall.through=Le cas précédent du "switch" ne contient pas de break, return, yield, throw ou continue.
+fall.through.last=Le dernier cas du "switch" ne contient pas de break, return, yield, throw ou continue.
```

Comment:
why changes are only in French localization? Other languages dont need it?

Reply:
Just found out Deutsch needs also, will amend. Except for two languages, translation is based on "Fall through from previous branch of the switch statement" so don't need it. Examples follow:

* [es] Caída desde la etiqueta anterior en la sentencia switch.
* [pt] Continuação (fallthrough) vindo da ramificação anterior da instrução switch.
* [tr] ''switch'' ifadesinin bir önceki durumundan aşağı düşme mevcut (''break'' kullanılmamış olabilir).

Reply:
Resolved in b2c18a8c08cddd27455d3520f4cb2d060c1a0041